flows/coordinatematch/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .coordinatematch import CoordinateMatch # noqa
```

Comment:
Be specific on which warnings you are disabling. Never use the "disable all warnings" unless there is a REALLY good reason for it.

Reply:
These are error F401 but we should ignore F401 by default on all __init__.py files.